app/server/appsmith-server/src/main/java/com/appsmith/server/migrations/DatabaseChangelog.java

```diff
@@ -1993,4 +1995,79 @@ public void changeAppLayoutTypeDefinition(MongoOperations mongoOperations, Mongo
         }
     }
 
+    @ChangeSet(order = "060", id = "update-mysql-postgres-mongo-ssl-mode", author = "")
```

Comment:
The `060` migrations is already there in release :)

```suggestion
    @ChangeSet(order = "061", id = "update-mysql-postgres-mongo-ssl-mode", author = "")
```